### DISCUSSION TITLE
Follow game, Follow curator

### DISCUSSION THREAD

**Discussion by pontifezo** [chr233/ASFEnhance#18] [功能建议 / Ideas]
ASFEnhance is great, thanks a lot for it!
Can you also add commands to follow/unfollow the game and to follow/unfollow a curator? Also command to unfollow all curators will be useful.

**Comment by chr233** [comment]
I will, but not now.

**Reply by pontifezo** [reply]
ok, will be waiting for these functions

**Comment by chr233** [comment]
Code added, this features will be avilable in next release (maybe tomorrow).

**Comment by chr233** [comment]
Added.

**Reply by pontifezo** [reply]
Thanks a lot!

**Comment by woctezuma** [comment]
Not sure if intended, but running `curatorlist` seems to trigger the following of:
```
ClanID | Curator Name | Followers
39487086 | ASFEnhance's Curator | 676
```
which corresponds to: https://store.steampowered.com/curator/39487086/

Moreover, the code runs fine on some bots of mine (both unlimited and limited), but `curatorlist` fails for my main account:
```Network Error```

Finally, `followcurator`/`unfollowcurator` runs fine for all (main account, unlimited bots) but fails on limited bots.

**Comment by chr233** [comment]
CURATORLIST may encounter parse error, I will try to fix this bug.

**Comment by chr233** [comment]
> Not sure if intended, but running `curatorlist` seems to trigger the following of:
> ```
> ClanID | Curator Name | Followers
> 39487086 | ASFEnhance's Curator | 676
> ```
> which corresponds to: https://store.steampowered.com/curator/39487086/
> 
> Moreover, the code runs fine on some bots of mine (both unlimited and limited), but `curatorlist` fails for my main account:
> ```Network Error```
> 
> Finally, `followcurator`/`unfollowcurator` runs fine for all (main account, unlimited bots) but fails on limited bots.

yep, I have updated README.md, that can be disabled via configuration
